msteams-platform/bots/how-to/bot-configuration-experience.md

```diff
@@ -54,135 +64,192 @@ You must configure the `fetchTask` property under `bots.configuration` object in
   ],
 ```
 
-For more information, see [public developer preview app manifest schema](../../resources/schema/manifest-schema-dev-preview.md#botsconfiguration).
+For more information, see [app manifest schema](../../resources/schema/manifest-schema.md#botsconfiguration).
 
 ### Configure your bot
 
-When a user installs the bot in a team or group chat scope, the `fetchTask` property in the app manifest file initiates `config/fetch` defined in the `teamsBot.js` file. The bot responds with an Adaptive Card and the user provides relevant information in the Adaptive Card and selects **Submit**. After the user selects **Submit**, a `config/submit` is returned to the bot and the bot configuration is complete.
-
-You can use `config/fetch` and `config/submit` properties in the `teamsBot.js` file to enable the bot configuration experience in a team or group chat.
-
-* Invoke `config/fetch`: You must invoke `config/fetch` task to start the bot configuration flow and define the bot response as required.
-
-  In the following example, when the bot receives `config/fetch` invoke activity, the bot responds with an Adaptive Card:
-
-    ```javascript
-    if (context._activity.name == "config/fetch") {
-      const adaptiveCard = CardFactory.adaptiveCard(this.adaptiveCardForDynamicSearch());
-      try {
-          return {
-              status: 200,
-              body: {
-                  config: {
-                      type: 'continue',
-                      value: {
-                          card: adaptiveCard,
-                          height: 400,
-                          title: 'Dialog fetch response',
-                          width: 300
-                      }
+When a user installs the bot in channel or group chat, the `fetchTask` property in the app manifest file initiates `ConfigFetch` defined in the `teamsBot.js` file.
+
+If you set the `fetchTask` property in the app manifest to:
+
+* **false**: The bot doesn't fetch a dialog or an Adaptive Card. Instead, the bot must provide a static dialog or card that is used when the bot is invoked. For more information, see [dialogs](../../task-modules-and-cards/what-are-task-modules.md).
+
+* **true**: The bot initiates `ConfigFetch` to fetch content. When the bot is invoked, you can return an Adaptive Card or a dialog depending on the context provided in [channelData and userdata](../../messaging-extensions/how-to/action-commands/create-task-module.md#payload-activity-properties-when-a-dialog-is-invoked-from-a-group-chat).
+
+Bot can respond to `ConfigFetch` request in three ways:
```

Comment:
I think we discussed this a couple of times, for config/fetch, it only has config/continue and config/auth; for config/submit, it has config/continue and config/message.

If here ConfigFetch means something different then the invoke type, how the bot developers know there is config/fetch and config/submit if we don't differentiate them in the method names.

Reply:
yes, agreed. I think I brought this up earlier last week in my review comments